Source/Analytics/Events.Reporting/CaseReports/CaseReportReceived.cs

```diff
@@ -12,7 +12,7 @@ public class CaseReportReceived : IEvent
         public CaseReportReceived(Guid dataCollectorId, Guid healthRiskId,
             string origin, string message, int numberOfMalesUnder5, int numberOfMalesAged5AndOlder,
             int numberOfFemalesUnder5, int numberOfFemalesAged5AndOlder, double longitude,
-            double latitude, DateTimeOffset timestamp)
+            double latitude, DateTimeOffset timestamp, string region)
```

Comment:
Are we allowed to do this change? Or will we ever get a region from this event?

Reply:
Nope, shouldn't have been there – removing.